engine/spv_wallet_engine_suite_test.go

```diff
@@ -88,100 +81,51 @@ func (ts *EmbeddedDBTestSuite) TearDownTest() {
 //
 // NOTE: you need to close the client: ts.Close()
 func (ts *EmbeddedDBTestSuite) createTestClient(ctx context.Context, database datastore.Engine,
-	tablePrefix string, mockDB, mockRedis bool, opts ...ClientOps,
+	tablePrefix string, opts ...ClientOps,
 ) (*TestingClient, error) {
 	var err error
 
 	// Start the suite
 	tc := &TestingClient{
 		ctx:         ctx,
 		database:    database,
-		mocking:     mockDB,
 		tablePrefix: tablePrefix,
 	}
 
-	// Are we mocking SQL?
-	if mockDB {
-
-		// Create new SQL mocked connection
-		if tc.SQLConn, tc.MockSQLDB, err = sqlmock.New(
-			sqlmock.QueryMatcherOption(sqlmock.QueryMatcherEqual),
-		); err != nil {
-			return nil, err
+	// Load the in-memory version of the database
+	if database == datastore.SQLite {
```

Comment:
```suggestion
	if database == datastore.SQLite {
	     // Sanity check: Ensure the Postgres server is initialized
             if ts.PostgresServer == nil {
                return nil, ErrLoadServerFirst
             }
```

Reply:
This doesn't make sense (Or I misunderstood your suggestion) - because `SQLite` doesn't use PostgressqlServer at all.
- It's just a file-based DB, managed by `go-sqlite3`.